### PR TITLE
Add regression test

### PR DESCRIPTION
**Pull request by shinya7y**
Thank you for the great repository!
I would like to add regression tests to verify that this repo outputs the same images as the official repo.

For testing, I generated the image (v1_4_astronaut_rides_horse_plms_step50_seed42.png) using [CompVis 7b8c883](https://github.com/CompVis/stable-diffusion/commit/7b8c883b078024f68b56e862f247a64f9e282aac) (without watermark).

## Test 1

`bash .dev_scripts/test_regression_txt2img_v1_4.sh`

```
comparing the following two images
IMAGE1: .dev_scripts/images/v1_4_astronaut_rides_horse_plms_step50_seed42.png
IMAGE2: outputs/txt2img-samples/test_regression_txt2img_v1_4/samples/00004.png
0.0
```

`scripts/orig_scripts/txt2img.py` outputs exactly the same image.

## Test 2

`bash .dev_scripts/test_regression_txt2img_dream_v1_4.sh`

```
comparing the following two images
IMAGE1: .dev_scripts/images/v1_4_astronaut_rides_horse_plms_step50_seed42.png
IMAGE2: outputs/img-samples/test_regression_txt2img_v1_4/000005.42.png
0.0
```

`scripts/dream.py` (with `--full_precision`) outputs exactly the same image.

### CODE REVIEW

**Comment by dagf2101** [comment]
Just ran your tests and I get the following results:
test_regression_txt2img_v1_4.sh: 0.5042826334635416
test_regression_txt2img_dream_v1_4.sh: 1.084851582845052

Dont know why but I think you should at least add the --full_precision flag when calling the dream.sh

File: .dev_scripts/test_regression_txt2img_dream_v1_4.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# generate an image
+PROMPT_FILE=".dev_scripts/sample_command.txt"
+OUT_DIR="outputs/img-samples/test_regression_txt2img_v1_4"
+SAMPLES_DIR=${OUT_DIR}
+python scripts/dream.py \
+    --from_file ${PROMPT_FILE} \
+    --outdir ${OUT_DIR} \
+    --sampler plms
```
Comment (dagf2101) [inline .dev_scripts/test_regression_txt2img_dream_v1_4.sh:8]: Think you should add --full_precision here.

**Comment by lstein** [comment]
Thank you for the PR. I will be integrating the latest round of requests over this weekend.

File: .dev_scripts/test_regression_txt2img_dream_v1_4.sh (hunk 1 above)
Comment (shinya7y) [reply .dev_scripts/test_regression_txt2img_dream_v1_4.sh]: Thank you for pointing it out.

**Review by lstein — APPROVED** [review]
This looks fine. Nice addition to have.

**Review by dagf2101 — APPROVED** [review]

**Pull request merged**